src/fidesops/ops/models/privacy_request.py

```diff
@@ -117,6 +117,7 @@ class PrivacyRequestStatus(str, EnumType):
     """Enum for privacy request statuses, reflecting where they are in the Privacy Request Lifecycle"""
 
     identity_unverified = "identity_unverified"
+    requires_input = "requires_input"
```

Comment:
Will be implemented in the followup PR when we actually wire up the ManualWebhook.  Privacy requests that still need manual input from a manual_webhook will be placed in a "requires_input" state.  

As an aside, we're getting a lot of privacy request statuses, and most statuses have specific statuses that they transition to next, and ones in which they should never transition to. It might be nice to formalize this at some point with some sort of state machine.